subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/GitAttributesGenerator.java

```diff
@@ -33,7 +33,10 @@ public void generate(InitSettings settings) {
                 writer.println("#");
                 writer.println("# https://help.github.com/articles/dealing-with-line-endings/");
                 writer.println("#");
-                writer.println("# These are explicitly windows files and should use crlf");
+                writer.println("# Linux start script should use lf");
```

Comment:
Added this in front of entry for `*.bat` because it is more specific and therefore it is less likely that users will edit this line.